src/gui/sdlmain.cpp

```diff
@@ -4170,9 +4170,7 @@ static void messages_add_sdl()
 	        "\n"
 	        "  --erasemapper            Delete the default mapper file.\n"
 	        "\n"
-	        "  --securemode             Enable secure mode. The [config] and [autoexec] sections\n"
-	        "                           of the loaded configurations will be ignored, and\n"
-	        "                           commands such as MOUNT and IMGMOUNT are disabled.\n"
+	        "  --securemode             Enable secure mode: disables the MOUNT and IMGMOUNT commands.\n"
```

Comment:
Stylistic suggestion:

"Enable secure mode; this disables..."

At least "enables" should be "enable"; I've normalised all descriptions to use imperative mood which IMO is the preferred succinct way to describe options, settings, etc.

Reply:
Now:

```c++
"  --securemode    Enable secure mode by disabling the MOUNT and IMGMOUNT\n"
"                  commands.\n"
```